internal/controller/controller.go

```diff
@@ -103,6 +103,11 @@ func Run(
 		return
 	}
 
+	if agentTokenIdentity.ClusterUUID == "" {
+		logger.Error("Detected unclustered agent token, a clustered agent token is required")
```

Comment:
Let's give folks a breadcrumb:

```suggestion
		logger.Error("Detected unclustered agent token, please upgrade to a clustered agent token: https://buildkite.com/organizations/~/clusters/~/tokens")
```

I'll make this URL work.